api/Navigator.json

```diff
@@ -320,15 +331,6 @@
             },
             "samsunginternet_android": {
               "version_added": null
-            },
-            "uc_android": {
-              "version_added": null
-            },
-            "uc_chinese_android": {
-              "version_added": null
-            },
-            "webview_android": {
-              "version_added": "40"
             }
```

Comment:
`webview_android` should be at the end of the `support` block (issues #1662 and #398)

/cc @jpmedley

Reply:
Thanks for the review, done!